paper.md

```diff
@@ -167,17 +171,16 @@ defined as the top-right pixel among the four central pixels of the image - a
 change motivated by the new default FT-based methods for image operations. The 
 center convention is unchanged for odd-size images (central pixel).
 
-Finally, a total of nine jupyter notebook tutorials covering most of the 
-available features in VIP were implemented. These tutorials illustrate (i) how 
-to load and post-process an ADI dataset (quick-start tutorial); (ii) how to 
-pre-process ADI and IFS datasets; (iii) how to model and subtract the stellar 
-halo with ADI-based algorithms; (iv) how to calculate metrics such as the S/N 
-ratio [@Mawet:2014], STIM maps [@Pairet:2019] and contrast curves; (v) how to 
-find the radial separation, azimuth and flux of a point source; (vi) how to 
-create and forward model scattered-light disk models; (vii) how to post-process 
-IFS data and infer the exact astro- and photometry of a given point source; 
-(viii) how to use FT-based and interpolation-based methods for different image 
-operations, and assess their respective performance; and (ix) how to use the 
+Finally, a total of eight jupyter notebook tutorials covering most of the 
```

Comment:
there are nine tutorials (my local version will be pushed soon)

Reply:
my bad, i meant to revert this